spec/requests/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Multiple test cases are failing. Kindly ensure that **all** integration test cases are passing. I also recommend that you include instructions on how to run test cases for your app in your ReadMe.

Failing tests:
![image](https://user-images.githubusercontent.com/12745474/193059363-4a43d1e4-e037-46a8-a549-6b1cd496425c.png)